website/plugins/proto-messages/index.js

```diff
@@ -1,7 +1,8 @@
-const visit = require("unist-util-visit");
+// Copyright 2022 Protosaurus Authors
```

Comment:
We can ts-ified later?

Reply:
Sure! I'll add a TODO. I'm unsure if Docusaurus plugins support TypeScript, but in any case, we can create a prebuild script to always `tsc` the `.ts` files.